radicle-cob/src/backend/git/change.rs

```diff
@@ -110,8 +110,15 @@ impl change::Storage for git2::Repository {
             history_type,
         };
 
-        let revision = write_manifest(self, &manifest, &contents)?;
+        let (revision, blob_oids) = write_manifest(self, &manifest, &contents)?;
         let tree = self.find_tree(revision)?;
+        let contents = blob_oids
+            .into_iter()
+            .zip(contents)
+            .map(|(oid, op)| (oid.into(), op))
+            .collect::<Vec<_>>();
+        // SAFETY: We know it's not empty because it was created from a `NonEmpty`.
+        let contents = NonEmpty::from_vec(contents).unwrap();
```

Comment:
```suggestion
        // SAFETY: We know it's not empty because it was created from a `NonEmpty`.
        let contents = NonEmpty::collect(blob_oids
            .into_iter()
            .zip(contents)
            .map(|(oid, op)| (oid.into(), op))).unwrap();
```